Taggers/python/flashggHHWWggTag_cfi.py

```diff
@@ -54,27 +53,42 @@
                                     TransverseImpactParam = cms.double(0.02),
                                     LongitudinalImpactParam = cms.double(0.2),
-                                    deltaRPhoElectronThreshold = cms.double(0.4), # was 1 
-                                    deltaMassElectronZThreshold = cms.double(10.),                                  
+                                    deltaRPhoElectronThreshold = cms.double(0.4), # was 1
+                                    deltaMassElectronZThreshold = cms.double(10.),
+                                    deltaMassElectronZ_FL_Threshold = cms.double(5.),
                                     electronEtaThresholds=cms.vdouble(1.4442,1.566,2.5),
                                     nonTrigMVAThresholds = cms.vdouble(0.913286,0.805013,0.358969),
                                     nonTrigMVAEtaCuts = cms.vdouble(0.8,1.479,2.5),
                                     electronIsoThreshold = cms.double(0.15),
                                     electronNumOfHitsThreshold = cms.double(1),
                                     useElectronMVARecipe = cms.bool(False),
-                                    useElectronLooseID = cms.bool(True),                                    
+                                    useElectronLooseID = cms.bool(True),
                                     rhoTag = cms.InputTag('fixedGridRhoFastjetAll'),
                                     RECOfilters = cms.InputTag('TriggerResults::RECO'),
                                     PATfilters = cms.InputTag('TriggerResults::PAT'),
                                     FLASHfilters = cms.InputTag('TriggerResults::FLASHggMicroAOD'),
                                     # bTag = cms.string(flashggDeepCSV),
-                                    # btagThresh = cms.double(100)     # no btag (Save all btags < 100)                                 
+                                    # btagThresh = cms.double(100)     # no btag (Save all btags < 100)
                                     btagThresh = cms.double(0.45),
-                                    doHHWWggTagCutFlowAnalysis = cms.bool(False), # save events for cut flow analysis 
+                                    doHHWWggTagCutFlowAnalysis = cms.bool(False), # save events for cut flow analysis
                                     doHHWWggNonResAnalysis = cms.bool(False),
-                                    doHHWWggFHptOrdered = cms.bool(False), # for FH final state, choose four leading pT jets as four jets    
-                                    doHHWWggDebug = cms.bool(False),
-                                    HHWWggAnalysisChannel = cms.string("SL") # final state analysis to run. SL by default. Can be SL, FL, or FH 
+                                    doHHWWggFHptOrdered = cms.bool(False), # for FH final state, choose four leading pT jets as four jets
+                                    doHHWWggDebug = cms.bool(True),
```

Comment:
should have debug false by default to avoid unnecessary print outs, especially when running over entire data sets as this will massively slow down the jobs. Should instead set to true with the flag doHHWWggDebug